src/l500/l500-color.cpp

```diff
@@ -120,7 +120,22 @@ namespace librealsense
             launch_perf_mode = RS2_HOST_PERF_DEFAULT;
 #endif
 
-            color_ep->register_option(RS2_OPTION_HOST_PERFORMANCE, std::make_shared<librealsense::float_option_with_description<rs2_host_perf_mode>>(option_range{ RS2_HOST_PERF_DEFAULT, RS2_HOST_PERF_COUNT - 1, 1, (float) launch_perf_mode }, "Optimize based on host performance, low power low performance host or high power high performance host"));
+            //
+            // Registering RS2_OPTION_HOST_PERFORMANCE with initial host performance mode
+            // also define the TPROC USB granularity settings for each endpoint on the depth sensor at each performance mode
+            // host performance mode defined in rs2_host_perf_mode
+            // settings values are validated through many experiments, change with care
+            //
+            // Endpoint           RS2_HOST_PERF_DEFAULT    RS2_HOST_PERF_LOW     RS2_HOST_PERF_HIGH
+            //   5 (color)                7 KB                32 KB                  7 KB
```

Comment:
The default values are the actual defaults set in the FW? I.e., the default may be different than both LOW and HIGH?